.travis.yml

```diff
@@ -25,30 +25,28 @@ before_install:
   - if [ "$TRAVIS_SECURE_ENV_VARS" == false ] ; then echo "OpenEye license will not be installed in pull request."; fi
 
 install:
-  - source devtools/travis-ci/install.sh
-  - export PYTHONUNBUFFERED=true
-
-script:
   # Create a test environment
   - conda create --yes -n test python=$python
   # Activate the test environment
-  - source activate test
-  - conda install --yes --quiet pip
+  - conda activate test
   # Build the recipe
   - conda build devtools/conda-recipe
   # Install the package
-  - conda install --yes --use-local ${PACKAGENAME}
+  - conda install --use-local ${PACKAGENAME}
   # Install testing dependencies
-  - conda install --yes --quiet nose nose-timer
+  - conda install --quiet nose nose-timer pytest
```

Comment:
Because we're installing `nose` here, we can probably get away with removing the `tests` section. The two should be redundant. Personally, we can just get rid of nose altogether and convert all the `@attr('slow')` and `@skipIf` decorators to `@pytest.mark.skip` and `@pytest.mark.skipif`.